introduction.md

```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
 
 * Are computers and robots morally responsible for their actions? 
 
+### Autonomous Machines
+
+* Can software possess autonomy?
+
+* Is autonomy a subjective or objective quality? 
+
+* Is there a need to change our definition of autonomy?
+
+### Memetics: replication and curation
+
+* How can the intellectual framework of memetics be applied to understand modern information technology?
+
+#### Economics and Heteromation
+
+* Is there an unexplained interdependence between technology and economics (which we are only slowly uncovering)?
```

Comment:
Surely the two _are_ interdependent. Economic activity relies on technology: assembly lines, word processors, satellites, etc. And technologies are developed to address or create demand. 

What is left to uncover?